mbuild/tests/test_compound.py

```diff
@@ -68,6 +68,17 @@ def test_save_forcefield_with_file(self, methane):
                          forcefield_files=get_fn('methane_oplssaa.xml'),
                          overwrite=True)
 
+    @pytest.mark.skipif(not has_foyer, reason="Foyer is not installed")
+    def test_save_forcefield_with_file_foyerkwargs(self, methane):
+        exts = ['.hoomdxml']
+        foyerkwargs = {'assert_improper_params': True}
+        with pytest.raises(Exception):
+            for ext in exts:
+                methane.save('lythem' + ext,
+                             forcefield_files=get_fn('methane_oplssaa.xml'),
```

Comment:
Since this file doesn't exist, won't it fail whether or not `foyerkwargs` is properly passed?

Reply:
Are you referring to `methane_oplsaa.xml`?